irohad/validation/impl/stateless_validator_impl.cpp

```diff
@@ -44,7 +44,6 @@ namespace iroha {
       ts64_t now = time::now();
 
       // tx is not sent from future
-      // TODO 06/08/17 Muratov: make future gap for passing timestamp, like with old timestamps IR-511 #goodfirstissue
```

Comment:
I moved it to the new stateless validation. This TODO is still not fixed. It was decided not to keep both comments. It will be fixed in new validator, not in this obsolete one